res/values/strings.xml

```diff
@@ -382,6 +383,10 @@
     <string name="pref_watch_mvbox_folder">Watch DeltaChat folder</string>
     <string name="pref_auto_folder_moves">Automatic Moves to DeltaChat folder</string>
     <string name="pref_auto_folder_moves_explain">Chat conversations are moved to avoid cluttering the Inbox folder</string>
+    <string name="pref_email_interaction_title">Email interaction</string>
+    <string name="pref_show_emails">Show classic emails</string>
```

Comment:
huh, shouldn't this be the "no" option and is more like "only show chat mails"?

Reply:
"off" is no separate string but a standard string currently.

do you suggest to name the option sth. as "Show classic emails? - No, chats only|For accepted contacts|all" ?